src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/components/base/DropdownComponent.tsx

```diff
@@ -15,6 +15,7 @@ export interface IDropdownProps {
   isValid?: boolean;
   optionsId: string;
   readOnly: boolean;
+  preselectedOptionIndex: number;
```

Comment:
This prop is optional. Make it so in the interface.

This will also fix the failing test.

```suggestion
  preselectedOptionIndex?: number;
```